src/Skipper/Skipper/SkipSkipper.php

```diff
@@ -35,4 +40,45 @@ public function doesMatchSkip(object | string $checker, string $filePath, array
 
         return false;
     }
+
+    /**
+     * @param list<Comment> $comments
+     */
+    public function doesMatchComments(object | string $checker, string $filePath, array $comments): bool
+    {
+        if ($comments === []) {
+            return false;
+        }
+        $currentRuleFullName = is_object($checker) ? $checker::class : $checker;
+        $currentRuleName = basename(str_replace('\\', '/', $currentRuleFullName));
+        foreach ($comments as $comment) {
+            $commentLines = explode("\n", $comment->getText());
```

Comment:
use `NewLineSplitter::split()` instead for windows support 

https://github.com/rectorphp/rector-src/blob/a22c3b9ff9f1889902ec42ccd3f4e3c15ac42a4b/src/Util/NewLineSplitter.php#L20-L23

Reply:
Updated